domain/src/main/java/com/architectcoders/aacboard/domain/use_case/dashboard/get/GetMainDashboardUseCase.kt

```diff
@@ -0,0 +1,22 @@
+package com.architectcoders.aacboard.domain.use_case.dashboard.get
+
+import com.architectcoders.aacboard.domain.data.dashboard.DashboardWithCells
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.flow.flatMapLatest
+
+class GetMainDashboardUseCase(
+    private val getPreferredDashboardIdUseCase: GetPreferredDashboardIdUseCase,
```

Comment:
Mhhhh.... very interesting 🤔 Do you think it is better this way instead of doing the same in the repository? The eternal doubt if this logic should be in the use case or in the repository.

Reply:
I have moved the logic to the repository